### ISSUE TITLE
Wrong smart-selects tamplate rendering

### ISSUE THREAD

**Issue by SalahAdDin** [digi604/django-smart-selects#120]
Hi guys

I want report this bug:
![seleccion_130](https://cloud.githubusercontent.com/assets/5034215/13032132/3baba19e-d2b5-11e5-92c7-8622919c0985.png)

As you can see the rendered script hasn't take `chainedfk` script and initialize it, because, well, if you see the script block, can't find any smart-selects script, inclusive any jQuery url.
This is a front-end template problem, in django-admin there is other problem that i report here, and this problem appears in django debug mode, i'd tested this in production.

I could solve this problem partially adding they in the template:
```html
{% block extra_css %}
    <!-- TODO: Delete this if Smart-Selects solve this problem -->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
<script src="{% static 'smart-selects/admin/js/chainedfk.js' %}"></script>
{% endblock %}
```

This code solve this bug, but the problem have to solved by smart-selects.

In addition i make a PR solving partially two problems with Smart-Selects, django-admin js script folders and JQUERY_URL, updating the URL i could solve a part of report problem.

**Comment by SalahAdDin** [comment]
Adding missin folder: https://github.com/digi604/django-smart-selects/issues/117

**Comment by jimmykobe1171** [comment]
@SalahAdDin could you have a try on the version of branch master? I have fixed some Jquery issues on Django 1.9.

**Comment by SalahAdDin** [comment]
No, i'm using pip package.

**Comment by julianx** [comment]
I was able to fix this using <script src="{% static 'smart-selects/admin/js/chainedfk.js' %}"></script> , as suggested by Zee in http://stackoverflow.com/questions/35338665/django-smart-selects-doesnt-work-in-templates/35772760

**Comment by SalahAdDin** [comment]
@jimmykobe1171 who is mantaining this package?

**Comment by julianx** [comment]
@SalahAdDin I think we (community) are in charge for the time being. Time to get my hands dirty :)

**Comment by jimmykobe1171** [comment]
oh guys, I am sorry, I was busy these months. I have much time now and will try to merge your fixs. I am happy to see your contribution！

**Comment by SalahAdDin** [comment]
@julianx yes, That's the attitude. 👍

**Comment by blag** [comment]
Should be fixed by #137. Closing. Feel free to reopen if it isn't.

**Comment by gdrio** [comment]
@julianx  your answer is correct but you have to take into consideration the following order:
script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
<script src="{% static 'smart-selects/admin/js/chainedfk.js' %}"></script>
